ivy/functional/ivy/creation.py

```diff
@@ -285,13 +285,66 @@ def full_like(
     but this function is *nestable*, and therefore also accepts :code:`ivy.Container`
     instances in place of any of the arguments.
 
-    Examples
-    --------
-    >>> x = ivy.array([1, 2, 3, 4, 5, 6])
-    >>> fill_value = 1
-    >>> y = ivy.full_like(x, fill_value)
-    >>> print(y)
-    ivy.array([1, 1, 1, 1, 1, 1])
+    Functional Examples
+    -------------------
+    With int datatype:
+
+        >>> import ivy
+        >>> x = ivy.array([1, 2, 3, 4, 5, 6])
+        >>> fill_value = 1
+        >>> y = ivy.full_like(x, fill_value)
+        >>> print(y)
+        ivy.array([1, 1, 1, 1, 1, 1])
+        >>> fill_value = 0.000123
+        >>> x = ivy.ones(5)
+        >>> y = ivy.full_like(x , fill_value)
+        >>> print(y)
+        ivy.array([0.000123, 0.000123, 0.000123, 0.000123, 0.000123])
+
+    with float datatype:
+
+        >>> import ivy
+        >>> x = ivy.array([1.0, 2.0, 3.0, 4.0, 5.0, 6.0]) # for filling values with floating datatype the array is required to be of same dtype (float in this case)
+        >>> fill_value = 0.000123
+        >>> y = ivy.full_like(x, fill_value)
+        >>> print(y)
+        ivy.array([0.000123, 0.000123, 0.000123, 0.000123, 0.000123, 0.000123])
+
+    with torch.Tensor input:
+
+        # ivy.full_like will give you output as ivy array
+        >>> import ivy
+        >>> import torch
+        >>> x = torch.Tensor([1.0, 2.0, 3.0, 4.0, 5.0, 6.0])
+        >>> fill_value = 0.001
+        >>> y = ivy.full_like(x, fill_value)
+        >>> print(y)
+        ivy.array([0.001, 0.001, 0.001, 0.001, 0.001, 0.001])
+
+    With ivy.NativeArray input:
+
+        >>> x = ivy.native_array([3.0, 8.0])
+        >>> fill_value = 0.000123
+        >>> y = ivy.full_like(x,fill_value)
+        >>> print(y)
+        ivy.array([0.000123, 0.000123])
+        >>> x2 = ivy.native_array([[3., 8., 2.],
+                                    [2., 8., 3.]])
+        >>> y = ivy.full_like(x2 , fill_value)
+        >>> print(y)
+        ivy.array([[0.000123, 0.000123, 0.000123],
+       [0.000123, 0.000123, 0.000123]])
+
+    Instance Method Examples:
+    ----------
+
+    With ivy.Array input:
+        >>> x = ivy.array([1, 2, 3, 4, 5, 6])
+        >>> fill_value = 1
+        >>> y = x.full_like(fill_value)
+        >>> print(y)
+        ivy.array([1, 1, 1, 1, 1, 1])
+
```

Comment:
please add ivy.Container instance method examples

Reply:
Hi @juliagsy I've tried to do full_like function with ivy.Container but it's not working there is issue with function wrapping ig

I've posted the error log here https://github.com/unifyai/ivy/issues/2120#issuecomment-1191468184

Reply:
got it